modules/ROOT/pages/environments.adoc

```diff
@@ -77,9 +78,9 @@ If your organization administrator provides multiple xref:managing-api-clients.a
 
 To associate a client provider with an existing environment:
 
-. Log in to Anypoint Platform using an account that has the organization administrator role.
-. Navigate to *Access Management*.
-. In the menu on the left, click *Environments*.
+include::partial$include-nav-steps-org-admin.adoc[]
+
+. In the Access Management navigation menu, click *Environments*.
```

Comment:
You can create a partial with this, too :)

Reply:
I could. I might revisit these nav substeps with Teams.